server/database/queries/filter/patchFilter.js

```diff
@@ -0,0 +1,11 @@
+const dbconnection = require('../../config/db_connection');
+
+exports.patchFilter = (skills, offerType, memberId) => {
+  const newSkills = skills.map(skill => (JSON.stringify(skill).toString()));
+  const newOfferTypes = offerType.map(offertype => (JSON.stringify(offertype).toString()));
```

Comment:
@AhmedAl-Almi I have a very minor refactoring suggestion: Move the `map(offertype => (JSON.stringify(offertype).toString()))` into a helper function, and call that function in the above two lines, since it's the same code happening twice on line 4 and 5.
It might seem like a waste of time, but it's a good idea to remove the duplicated code because then you can change the function in the future without the danger of forgetting to update it in both places (which does happen often - you go back into your code later to fix a bug, but you only fix it in one place and forget to fix it in the other place too)

Reply:
Thank you Tom for you valuable suggestion,
I already deleted these lines coz no need for them.

Reply:
Ah ok!